### ISSUE TITLE
A11y_SQLAzureDataStudio_AddaSQLServerBigDataCluster-Controller_Addnewcontroller_ScreenReader:The label of the edit box is announced as heading and item.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9230]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#MacOs;#MAS1.3.1;#VoiceOver;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

**Additional Details:**
Mac OS:10.15.3
MAS References:MAS1.3.1
Screen reader: Voiceover.

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Connections from left menu.
4. Select Add a SQL Server Big Data Clusters.
5. Navigate to + button & hit enter.
6. Launch voice over and navigate to the lable of the 1st edit box using control+ command +(left/right)arrow key

**Actual:**
On navigating to the labels of the edit first and second edit box using control+ command +(left/right)arrow key the label is announced as heading level 2 item.

**Same issue is observed in the following page:**
1.Import Bacpac Settings
2.Export Bacpac Settings

**Expected:**
The label should be just announced by its name and not heading level and item.

**User Impact:**
Screen reader users will receive wrong information of the label as structuring as well as they will consider it as an item.

**Attachment for referance:**
![image](https://user-images.githubusercontent.com/24580570/74928015-af88d400-53fe-11ea-9ea1-8198a9e59309.png)

**Comment by alanrenmsft** [comment]
we can easily change it to be just a div, but need to know when to use heading

**Comment by alanrenmsft** [comment]
lets not use h2 here, i will also email the accessibility team asking for guideline.

**Comment by Yashi1201** [comment]
As "SQL Server Big Data Cluster" feature has been removed
**GithubTags:**#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-MAC-Feb2020;
